# find_clustering_graph_max_node, graph, sequence_alignment_diagram, sequence_beam_search, lev_path, align_seq
Cluster subgraph: A subgraph formed by connecting similar DNA sequences by edges in a graph based on sequence similarity. 
Maximum node subgraph: A subgraph centered on the node with the largest degree (the node with the most connections to other nodes) in a clustered subgraph, which represents the key information of the cluster. 
Diverse beam search: constructs a diverse beam search graph and searches for the optimal solution of multiple candidate path sets, which is used to improve the accuracy of MSA. 
Reads (k-mer): In DNA sequencing, the segmentation of a DNA sequence into fixed-length fragments for subsequent sequence analysis and assembly. 
De Bruijn graph: used to represent overlapping k-mer of DNA sequences. Path search: the process of finding a path from a start point to an end point in a graph, used to solve problems such as path selection. 
Path selection: in sequence assembly, the process of selecting the best path from possible paths according to certain rules and algorithms to reconstruct the complete DNA sequence.
